visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -253,7 +253,7 @@ export interface Node {
 	z0: number
 	y0: number
 	isLeaf: boolean
-	deltas: KeyValuePair
+	deltas?: KeyValuePair
```

Comment:
This is an important change as it's not guaranteed to have deltas.

Reply:
Exactly. As far as I understood the code, the delta property is only calculated and decorated, when we're in the delta mode. In this case, we check the two maps `reference` and `comparison`, determine the difference between nodes and add that as the property `delta`. Am I correct?